topsort-api-v2.yml

```diff
@@ -0,0 +1,603 @@
+
+      ## Event Response
+      <SchemaDefinition schemaRef="#/components/schemas/EventResponse" />
+
+      ## Geotargeting
+      <SchemaDefinition schemaRef="#/components/schemas/GeoTargeting" />
+
+      ## Impression
+      <SchemaDefinition schemaRef="#/components/schemas/Impression" />
+
+      ## Impression Event
+      <SchemaDefinition schemaRef="#/components/schemas/ImpressionEvent" />
+
+      ## Impression Response
+      <SchemaDefinition schemaRef="#/components/schemas/ImpressionResponse" />
+
+      ## Placement
+      <SchemaDefinition schemaRef="#/components/schemas/Placement" />
+
+      ## Product
+      <SchemaDefinition schemaRef="#/components/schemas/Product" />
+
+      ## Purchase Event
+      <SchemaDefinition schemaRef="#/components/schemas/PurchaseEvent" />
+
+      ## Purchase Item
+      <SchemaDefinition schemaRef="#/components/schemas/PurchaseItem" />
+
+      ## Search Query
+      <SchemaDefinition schemaRef="#/components/schemas/SearchQuery" />
+
+      ## Session
+      <SchemaDefinition schemaRef="#/components/schemas/Session" />
+
+      ## Topsort Error
+      <SchemaDefinition schemaRef="#/components/schemas/TopsortError" />
+
+      ## Winner
+      <SchemaDefinition schemaRef="#/components/schemas/Winner" />
+
+security:
+  - BearerAuth: []
+
+paths:
+  /auctions:
+    post:
+      tags:
+        - Auctions
+      summary: Creates a new auction
+      operationId: createAuction
+      requestBody:
+        description: >
+          The information describing what will be auctioned.
+          Topsort will run an auction for each batched auction request, for which products, brands and/or vendors' bids will compete against each other.
+          Provided fields are additive, in the sense that only bids that match all filtering fields are considered (e.g. in the example below, only bids for products in the c_yogurt category that match the "Noosa Peach" search string will participate in the acution).
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                auctions:
+                  type: array
+                  items:
+                    oneOf:
+                      - $ref: '#/components/schemas/AuctionRequestSponsoredListings'
+                      - $ref: '#/components/schemas/AuctionRequestBannerAds'
+                    discriminator:
+                      propertyName: type
+                      mapping:
+                        listings: '#/components/schemas/AuctionRequestSponsoredListings'
+                        bannerAds: '#/components/schemas/AuctionRequestBannerAds'
+        required: true
+      responses:
+        201:
+          description: >
+            The auction results.
+            The list of winners will contain at most `slots` entries per auction.
+            It may contain fewer or no entries at all if there aren't enough products with usable bids.
```

Comment:
In the definition of usable there's no mention at all about being above the reserve price.